src/converters/typescript/string-format.js

```diff
@@ -125,6 +273,6 @@ function generateTSStringInterfaceCatergory(tokens, categoryInterfaceName) {
   return [
     `export interface ${categoryInterfaceName} {`,
     ...tokens.map((token) => `${token.name}: string;`),
-    '}',
+    '} \n',
```

Comment:
Maybe we should check that this works also in Windows. Because sometimes Windows requires to use `\r\n` instead to get it right, but UNIX systems use this `\n`.

Reply:
didn't encounter any issues building or using on windows.